src/components/header/productsDropDown/productsDropDown.tsx

```diff
@@ -25,13 +25,13 @@ export default function ProductsDropDown({ isProductsDropdownVisible, authData }
         authData?.username && styles.loggedIn,
       )}
     >
-      <button type="button" className={styles.linkItem} onClick={() => handleNavigation(`${PATHS.PRODUCTS}/pc`)}>
+      <button type="button" className={styles.linkItem} onClick={() => handleNavigation(PATHS.PC)}>
```

Comment:
No issue, but recommendation.
Here 3 similar buttons and different only path & text content so possible to improve DX (developer experience)
```tsx
[
    { path: PATHS.PS5, text: 'Playstation 5'},
    ...
].map(a=>{
   return <button ... onClick={a.path}>{a.text}</button>
})
```